docs/source/index.rst

```diff
@@ -96,6 +96,7 @@ Daml Documentation
    tools/visual
    concepts/interoperability
    tools/non-repudiation
+   ops/connect/index
```

Comment:
the reason this is failing is because you also need to update `docs/configs/pdf/index.rst`.

Reply:
Thanks for the help. I probably missed it. Where was this documented?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13052/commits/a147a340ab6771dab7284a3cc6061f6d096ba6d2

Reply:
not aware of any docs for that. Note that thanks to our amazing new release process you also need to duplicate those changes in the assembly repo which also has separate html & pdf index files.